build_scripts/build_usd.py

```diff
@@ -985,13 +985,20 @@ def InstallTIFF(context, force, buildArgs):
 
 def InstallPNG(context, force, buildArgs):
     macArgs = []
-    if MacOS() and Arm():
-        # ensure libpng's build doesn't erroneously activate inappropriate
-        # Neon extensions
-        macArgs = ["-DPNG_HARDWARE_OPTIMIZATIONS=OFF", 
-                   "-DPNG_ARM_NEON=off"] # case is significant
     with CurrentWorkingDirectory(DownloadURL(PNG_URL, context, force)):
-        RunCMake(context, force, buildArgs + macArgs)
+        if MacOS():
+            if Arm():
+                # ensure libpng's build doesn't erroneously activate inappropriate
+                # Neon extensions
+                macArgs = ["-DPNG_HARDWARE_OPTIMIZATIONS=OFF",
+                          "-DPNG_ARM_NEON=off"] # case is significant
+            PatchFile("CMakeLists.txt",
+                [('add_custom_target(gensym DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/libpng.sym")',
+                  'add_custom_target(gensym DEPENDS "${CMAKE_CURRENT_BINARY_DIR}/libpng.sym" genvers)'),
+                 ("add_custom_target(genfiles DEPENDS",
+                  "add_custom_target(genfiles DEPENDS gensym symbol-check")])
+
+        RunCMake(context, force, buildArgs)
```

Comment:
This is missing the "+ macArgs" that is needed to pass the Neon optimization args for Arm builds down to RunCMake().